cmd/allocator/main.go

```diff
@@ -97,6 +97,12 @@ func main() {
 		logger.WithError(err).Fatalf("cannot watch folder %s for secret changes", certDir)
 	}
 
+	watcherTLS, _ := fsnotify.NewWatcher()
```

Comment:
It looks like this does the same thing as on line 94 above, but the second return value from NewWatcher is an error, which we are ignoring. Code should never ignore errors.

See https://github.com/golang/go/wiki/CodeReviewComments#handle-errors

Reply:
Checked the errors in the creation of both watchers in the new PR (https://github.com/googleforgames/agones/pull/1638).